.github/workflows/tarpaulin.yml

```diff
@@ -0,0 +1,66 @@
+name: tarpaulin
+
+on:
```

Comment:
In general, the code coverage should be determined if Rust code was changed.  I also like to be informed about it when Rust dependencies are updated by Dependabot, so I left this configuration as I took it from Aeruginous.